action.yml

```diff
@@ -4,5 +4,35 @@ runs:
   using: "composite"
   steps:
       - name: "Process coverage data"
-        run:  pkg-ci-scripts/gather-coverage.sh
+        run: |
+         set -ex
+         GAPROOT=${GAPROOT-$HOME/gap}
```

Comment:
I wonder if `setup-gap` could export a global GAPROOT environment variable, so that all the other actions could reference it?

I never was comfortable with hardcoding a GAPROOT location everywhere; then again, it's not that bad as we control the environment.

But on the longterm, I guess we could instead let `setup-gap` install `gap` (or rather some shell script with that name which invokes the real GAP?) into a directory in the PATH. For the remaining cases that need it, well, one can figure it out from GAP code via `Filename(DirectoriesLibrary(""),"sysinfo.gap");` (and then remove `"/sysinfo.gap"` from the end).

Finally, I think that future GAP versions should have a `--get-gaproot` or `--sysinfopath` or `--get-primary-gaproot` command line option so that one can query this more easily for scripts.

Reply:
This sounds like a sensible approach, installing a `gap` script, and then using that entrypoint to GAP to obtain the GAPROOT.